vcpkg/reference/software-bill-of-materials.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Software Bill of Materials in vcpkg
+description: This article describes software bill of materials generated by vcpkg.
+author: data-queue
+ms.author: danshaw2
+ms.date: 1/19/2024
+ms.topic: reference
+---
+
+# Software Bill of Materials in vcpkg
+vcpkg generates a Software Bill of Materials (SBOM) based on the [Software Package Data Exchange (SPDX)](https://spdx.github.io/spdx-spec/v2.3/) specification. It tracks important information used to build a package, such as source origin, and aims to provide package consumers with software transparency and integrity. See this [blog post about SBOM and SPDX](https://devblogs.microsoft.com/engineering-at-microsoft/generating-software-bills-of-materials-sboms-with-spdx-at-microsoft/) for more information.
+
+vcpkg generates a SPDX file containing the SBOM information for each package that is installed. The files are located in `<installed_dir>/<triplet>/share/<package name>/vcpkg.spdx.json`. The installation directory depends on whether vcpkg is running on [manifest mode](../concepts/manifest-mode.md) or [classic mode](../concepts/classic-mode.md). Since a package can have different dependencies depending on the target platform, the generated files are separated by triplet as well.
```

Comment:
```suggestion
vcpkg generates a SPDX file containing the SBOM information for each package that is installed. The files are located in `<installed_dir>/<triplet>/share/<package name>/vcpkg.spdx.json`. The installation directory depends on whether vcpkg is running on [manifest mode](../concepts/manifest-mode.md) or [classic mode](../users/classic-mode.md). Since a package can have different dependencies depending on the target platform, the generated files are separated by triplet as well.
```